OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/IconProgressBar.java

```diff
@@ -89,6 +90,10 @@ public void setIndeterminateIconColor(@NonNull ColorStateList colorStateList) {
         }
     }
 
+    public void showProgress(boolean isShow) {
+        this.setVisibility(isShow ? View.VISIBLE : View.GONE);
```

Comment:
I think there is no need to update the generic implementation for the single usage only.